packages/core/src/pagination/pagination-number.element.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { baseStyles } from '@cds/core/internal';
+import { html, LitElement } from 'lit-element';
+import { styles } from './pagination-number.element.css.js';
+
+export class CdsPaginationNumber extends LitElement {
```

Comment:
If I am reading this correctly, this component is like the textual content of the pagination? Could it be something other than a number? If so, we may want to consider renaming it to `CdsPaginationContent` instead...

Reply:
I agree that `cds-pagination-number` could be a poorly named component. But I am also avoiding `cds-pagination-content` as users might assume they could place/project their content only within that component. They could actually place their content anywhere within the `cds-pagination` like this:

```html
<cds-pagination>
    Lorem Ipsum
    <cds-pagination-button action="prev"></cds-pagination-button>
    <cds-pagination-number>1 / 3<cds-pagination-number>
    <cds-pagination-button action="next"></cds-pagination-button>
    Bacon Ipsum
</cds-pagination>
```

But I still think we could/should definitely rename it differently than `cds-pagination`. 

Also before renaming it, I would like to step back and provide some context in why we need `cds-pagination-number`. The goal of `cds-pagination-number` is to provide a **default** wrapper style to the current/total page numbers. Users can apply custom styles and control the spacing of the current/total pages numbers while keeping the pagination items' symmetrical look. For Example:

```html
<cds-pagination>
    <cds-pagination-button action="prev"></cds-pagination-button>
    <cds-pagination-number cds-layout="horizontal m-x:lg">1 / 3<cds-pagination-number>
    <cds-pagination-button action="next"></cds-pagination-button>
</cds-pagination>
```

But most importantly, the component also allows us to provide a default input style:

```html
<cds-pagination>
    <cds-pagination-button action="prev"></cds-pagination-button>
    <cds-pagination-number><input type="text" value="1" size="1" /> / 3</cds-pagination-number>
    <cds-pagination-button action="next"></cds-pagination-button>
</cds-pagination>
```

So without `cds-pagination-number`, we both lose the symmetrical look and the ability to target that particular input for the current page.

I also brainstormed some ideas with Cory. One was to use `cds-input` without a wrapper component such as like `cds-pagination-number`. However, after some investigation, it might just complicate the API more. For example:


```html
<cds-pagination>
    <cds-pagination-button action="prev"></cds-pagination-button>
    <cds-input box control-width="shrink">
      <input type="text" value="1" size="1" aria-label="Edit current page" />
    </cds-input> / 3
    <cds-pagination-button action="next"></cds-pagination-button>
</cds-pagination>
```

And even with this API, we might need a wrapper component to keep the symmetry when the current and total page numbers are used together. Another consideration is what if users might want to use not just one but multiple `cds-input` within `cds-pagination`. All of them could get the current page number input style. But I will continue my investigation.